src/EditorFeatures/Test2/InlineParameterNameHints/CSharpInlineParamNameHintsTests.vb

```diff
@@ -0,0 +1,272 @@
+}
+                    </Document>
+                </Project>
+            </Workspace>
+
+            Await VerifyParamHints(input)
+        End Function
+
+        <WpfFact, Trait(Traits.Feature, Traits.Features.InlineParameterNameHints)>
+        Public Async Function TestObjectCreationParametersSimpleCase() As Task
+            Dim input =
+            <Workspace>
+                <Project Language="C#" CommonReferences="true">
+                    <Document>
+class A
+{
+    int testMethod(int x, object y)
+    {
+        return x;
+    }
+    void Main() 
+    {
+        $$testMethod({|x:(int)5.5|}, {|y:new object()|});
+    }
+}
+                    </Document>
+                </Project>
+            </Workspace>
+
+            Await VerifyParamHints(input)
+        End Function
+
+        <WpfFact, Trait(Traits.Feature, Traits.Features.InlineParameterNameHints)>
+        Public Async Function TestCastingANegativeSimpleCase() As Task
+            Dim input =
+            <Workspace>
+                <Project Language="C#" CommonReferences="true">
+                    <Document>
+class A
+{
+    int testMethod(int x, object y)
+    {
+        return x;
+    }
+    void Main() 
+    {
+        $$testMethod({|x:(int)-5.5|}, {|y:new object()|});
+    }
+}
+                    </Document>
+                </Project>
+            </Workspace>
+
+            Await VerifyParamHints(input)
+        End Function
+
+        <WpfFact, Trait(Traits.Feature, Traits.Features.InlineParameterNameHints)>
+        Public Async Function TestNegatingACastSimpleCase() As Task
+            Dim input =
+            <Workspace>
+                <Project Language="C#" CommonReferences="true">
+                    <Document>
+class A
+{
+    int testMethod(int x, object y)
+    {
+        return x;
+    }
+    void Main() 
+    {
+        $$testMethod({|x:-(int)5.5|}, {|y:new object()|});
+    }
+}
+                    </Document>
+                </Project>
+            </Workspace>
+
+            Await VerifyParamHints(input)
+        End Function
+
+        <WpfFact, Trait(Traits.Feature, Traits.Features.InlineParameterNameHints)>
+        Public Async Function TestMissingParameterNameSimpleCase() As Task
+            Dim input =
+            <Workspace>
+                <Project Language="C#" CommonReferences="true">
+                    <Document>
+class A
+{
+    int testMethod(int)
+    {
+        return 5;
+    }
+    void Main() 
+    {
+        $$testMethod();
+    }
+}
+                    </Document>
+                </Project>
+            </Workspace>
+
+            Await VerifyParamHints(input)
+        End Function
+
+        <WpfFact, Trait(Traits.Feature, Traits.Features.InlineParameterNameHints)>
+        Public Async Function TestDelegateParameter() As Task
+            Dim input =
+            <Workspace>
+                <Project Language="C#" CommonReferences="true">
+                    <Document>
+delegate void D(int x);
+
+    class C
```

Comment:
Indenting seems odd here since the classes are indented for no specific reason...

Reply:
That's just what happened when I copied and pasted the code in, fixed.